### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'message')

### ISSUE THREAD

**Issue by MalteKiefer** [immich-app/CLI#98]
Good day,

I would like to perform a bulk upload.
I am using the latest version of the CLI client.
However, when I want to perform the upload, I get the following message:

```
> .yarn/bin/immich upload --key XXXX --server https://pXXX/api -al -r Bilder -y                                                                           
Checking connectivity with Immich instance...
Server status: OK
Checking credentials...
Login status: OK
Successful authentication for user XXXX
Indexing local assets...
Indexing complete, found 887 local assets
Comparing local assets with those on the Immich instance...
A total of 887 assets will be uploaded to the server
Start uploading...
Upload Progress | ████████████████████████████████████████ | 100% || 677/677 || Current file [/home/XXX/Bilder/Urlaub/Österreich 2017/FB24D137-61BC-49DC-A7E5-745106B2C1D220170611_145933-1.jpeg]
Creating albums...
Upload Progress | ████████████████████████████████████████ | 100% || 10/10 || Current file [{filepath}]
Error reading input from user  TypeError: Cannot read properties of undefined (reading 'message')
    at /home/XXX/.config/yarn/global/node_modules/immich/bin/index.js:322:40
    at Generator.next (<anonymous>)
    at fulfilled (/home/XXX/.config/yarn/global/node_modules/immich/bin/index.js:29:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

```

**Comment by AirP0WeR** [comment]
Same issue for me.

**Comment by alextran1502** [comment]
Thank you, in the meantime, you can revert to back to the previous version to see if it still gives you the error `npm i immich@0.37.0`

With the error shown above, I assume it finished uploading but not putting those assets into the album, correct?

**Comment by AirP0WeR** [comment]
> Thank you, in the meantime, you can revert to back to the previous version to see if it still gives you the error
> 
> ```
> npm i -g immich@0.37.0
> ```
> 
> With the error shown above, I assume it finished uploading but not putting those assets into the album, correct?

This works for me? Thanks!

**Comment by alextran1502** [comment]
> Not for me, when I use this option:
> 
> ```
> -al
> ```
> 
> I get this error:
> 
> ```
> Error connecting to server - check server address and port
> ```
> 
> Without it is working

try `--album`

**Comment by alextran1502** [comment]
The issue reported should be fixed in v0.38.1 which is being built and deploy
